cupyx/scipy/spatial/_delaunay.py

```diff
@@ -0,0 +1,83 @@
+
+    Parameters
+    ----------
+    points : ndarray of floats, shape (npoints, ndim)
+        Coordinates of points to triangulate
+    furthest_site : bool, optional
+        Whether to compute a furthest-site Delaunay triangulation. This option
+        will be ignored, since it is not supported by CuPy
+        Default: False
+    incremental : bool, optional
+        Allow adding new points incrementally. This takes up some additional
+        resources. This option will be ignored, since it is not supported by
+        CuPy. Default: False
+
+    Attributes
+    ----------
+    points : ndarray of double, shape (npoints, ndim)
+        Coordinates of input points.
+    simplices : ndarray of ints, shape (nsimplex, ndim+1)
+        Indices of the points forming the simplices in the triangulation.
+        For 2-D, the points are oriented counterclockwise.
+    neighbors : ndarray of ints, shape (nsimplex, ndim+1)
+        Indices of neighbor simplices for each simplex.
+        The kth neighbor is opposite to the kth vertex.
+        For simplices at the boundary, -1 denotes no neighbor.0
+    vertex_to_simplex : ndarray of int, shape (npoints,)
+        Lookup array, from a vertex, to some simplex which it is a part of.
+        If qhull option "Qc" was not specified, the list will contain -1
+        for points that are not vertices of the tessellation.
+    convex_hull : ndarray of int, shape (nfaces, ndim)
+        Vertices of facets forming the convex hull of the point set.
+        The array contains the indices of the points belonging to
+        the (N-1)-dimensional facets that form the convex hull
+        of the triangulation.
+
+        .. note::
+
+           Computing convex hulls via the Delaunay triangulation is
+           inefficient and subject to increased numerical instability.
+           Use `ConvexHull` instead.
+    coplanar : ndarray of int, shape (ncoplanar, 3)
+        Indices of coplanar points and the corresponding indices of
+        the nearest facet and the nearest vertex.  Coplanar
+        points are input points which were *not* included in the
+        triangulation due to numerical precision issues.
+
+        If option "Qc" is not specified, this list is not computed.
+
+        .. versionadded:: 0.12.0
+    vertex_neighbor_vertices : tuple of two ndarrays of int; (indptr, indices)
+        Neighboring vertices of vertices. The indices of neighboring
+        vertices of vertex `k` are ``indices[indptr[k]:indptr[k+1]]``.
+    furthest_site
+        True if this was a furthest site triangulation and False if not.
+
+        .. versionadded:: 1.4.0
+    """
+
+    def __init__(self, points, furthest_site=False, incremental=False):
+        if points.shape[-1] != 2:
+            raise ValueError('Delaunay only supports 2D inputs at the moment.')
+
+        if furthest_site:
+            warnings.warn(
+                'furthest_site argument is not supported by CuPy. Ignoring')
```

Comment:
I think it is better to hard error in this situation. The user may not notice the warning and get wrong results